src/main/scala/com/databricks/labs/overwatch/pipeline/PipelineTargets.scala

```diff
@@ -154,6 +186,24 @@ abstract class PipelineTargets(config: Config) {
       config
     )
 
+    lazy private[overwatch] val jobRunsSnapshotTarget: PipelineTable = PipelineTable(
+      name = "job_runs_snapshot_bronze",
+      _keys = Array("job_id", "run_id", "Overwatch_RunID"),
+      config,
+      incrementalColumns = Array("Pipeline_SnapTS"),
+      statsColumns = "created_time, creator_user_name, job_id, run_id, Pipeline_SnapTS, Overwatch_RunID".split(", "),
```

Comment:
do all of these columns actually exist? Do you truly want to move all these to the front and have stats calculated on these columns? If so, no issue, but just wanted to check it's not another copy/paste issue.

Reply:
removed  `statsColumns`